autogen/main/variables.tf.tmpl

```diff
@@ -78,6 +78,12 @@ variable "master_authorized_networks" {
   default     = []
 }
 
+variable "gcp_public_cidrs_access_enabled" {
+  type        = bool
+  description = "Allow access through Google Cloud public IP addresses"
+  default     = false
```

Comment:
As `gcp_public_cidrs_access_enabled` is optional, I would suggest setting to `null` to preserve the current behavior when `gcp_public_cidrs_access_enabled` isn't set by a user:
```suggestion
  default     = null
```